### PR TITLE
adding-about-page

### PR DESCRIPTION
**Pull request by mccallum**
Adding about page

### CODE REVIEW

File: app/views/static/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    <h2>About Oregon Digital</h2>
+
+    <h3>The System</h3>
+
+    <p>A unique collaboration between the University of Oregon Libraries and Oregon State University Libraries &amp; Press, Oregon Digital is a <a href="http://projecthydra.org/">Hydra Project</a>. All code has been made publicly available on <a href="https://github.com/">Github</a>:</p>
+    <ul>
+      <li><a href="https://github.com/OregonDigital/oregondigital">Oregon Digital application</a></li>
+      <li><a href="https://github.com/OregonDigital/metadata-ingest-form">Metadata ingest forms</a></li>
+      <li><a href="https://github.com/OregonDigital/cdm2bag">Data transformations from legacy system</a></li>
```
Comment (wickr) [inline app/views/static/about.html.erb]: Probably worth adding csv2bag as well.

File: app/views/static/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    <h3>Metadata</h3>
+
+    <p>Oregon Digital operates under linked open data principles. Rather than prescribe a specific schema, we believe in description at the appropriate level with properties available in an open world context. Individual templates are created for easy entry for particular collections and groups of items.</p>
+
+    <ul>
+      <li><a href="https://docs.google.com/spreadsheets/d/1umwlp3wqJZjfLMig84IxX2S6qVZWjOdIbzTIxrcTg7o/edit?usp=sharing">Metadata Dictionary</a></li>
+      <li><a href="http://opaquenamespace.org/">Locally Hosted Terms and URIs</a></li>
+      <li><a href="http://oregondigital.org/copyright">Copyright Statements</a></li>
+      <li><a href="https://github.com/OregonDigital/opaque_ns/wiki">Metadata Principles</a></li>
```
Comment (wickr) [inline app/views/static/about.html.erb]: We should look at moving the metadata principles elsewhere, probably to the OD wiki. That repo is very old.

File: app/views/static/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    </ul>
```
Comment (wickr) [inline app/views/static/about.html.erb]: I'd link to the Controlled Vocabularies and Vocabularies pages as well.

File: app/views/static/about.html.erb [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+    <h4>Take-Down Policy</h4>
+
+    <p>Both Oregon State University Libraries &amp; Press and the University of Oregon Libraries strive to identify the copyright status and contact rights holders for items where permission has not been granted or an argument for <a href="http://fairuse.stanford.edu/overview/fair-use/">fair use</a> has not been determined. If you are a copyright holder and feel your rights have been violated through the provision of access, a request may be submitted to remove the item from Oregon Digital. The take-down policy also applies to digital objects that may contain personal data or culturally sensitive material.</p>
+
+    <p>Please send requests to Michaela Willi Hooper (michaela.willihooper@oregonstate.edu) with the subject line: Take-Down Request. In the email, include the following:</p>
```
Comment (vantuyls) [inline app/views/static/about.html.erb]: recommend making the email opaque so that it is not machine actionable and spammable.

**Comment by wickr** [comment]
Add link to About page to text block under Browse by Collection button.

Add link to About page and Vocabularies page in footer (but not Controlled Vocabularies).

**Comment by wickr** [comment]
Change contact from Michaela to Michael Boock.

Remove commented out pieces.

**Comment by mccallum** [comment]
I only added "About Oregon Digital" to the footers. Vocabularies and Controlled Vocabularies are linked from that About Oregon Digital.

**Pull request merged**